dist/licenses/LICENSE-prometheus-client-golang.txt

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Apache 2.0 license is not required to be copied. If NOTICE exists, you need to copy the content into `dist/NOTICE`.